changelog.d/pr-1672.change

```diff
@@ -0,0 +1 @@
+Polls: fetch polls event on m.poll.end
```

Comment:
This is supposed to be a bit more user friendly, maybe something along the lines of: "ensure polls are up to date after they get closed"